pupil-spa/src/app/login.userService.response.mock.json

```diff
@@ -24,5 +24,16 @@
     "jwt": {
       "token": "token"
     }
+  },
+  "pupil": {
+    "firstName": "Juliana",
+    "lastName": "Brewer",
+    "dob": "2 February 2009",
+    "checkCode": "17F27C48-3CA8-4E28-907A-6A72D982C342"
```

Comment:
i would add `uuid` in here under pupil, as that is now part of the payload.  just for completeness 👌